financial-connections/src/main/java/com/stripe/android/financialconnections/domain/IsNetworkingRelinkSession.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.financialconnections.domain
+
+import com.stripe.android.financialconnections.repository.CoreAuthorizationPendingNetworkingRepairRepository
+import javax.inject.Inject
+
+internal fun interface IsNetworkingRelinkSession {
```

Comment:
We have multiple screens where we want to know if we’re in a networking relink flow. This is easier to inject and test than using the full `CoreAuthorizationPendingNetworkingRepairRepository`.